app/Http/Controllers/Store/CentreController.php

```diff
@@ -193,21 +190,19 @@ private function getCentreRegistrationsSummary(array $centre_ids, $dateFormats =
                 }
             }
             // Add count of eligible kids
-            $row["Eligible Children"] = $eligibleKids;
+            $row['Eligible Children'] = $eligibleKids;
 
             // Add our kids back in
             $row = array_merge($row, $kids);
 
             // Set the last dates.
-            $row["Due Date"] = $due_date;
-            $row["Join Date"] = $reg->created_at ? $reg->created_at->format($dateFormats['join']) : null;
-            $row["Leaving Date"] = $reg->family->leaving_on ? $reg->family->leaving_on->format($dateFormats['leave']) : null;
+            $row['Due Date'] = $due_date;
+            $row['Join Date'] = $reg->created_at ? $reg->created_at->format($dateFormats['join']) : null;
+            $row['Leaving Date'] = $reg->family->leaving_on ? $reg->family->leaving_on->format($dateFormats['leave']) : null;
             // Would be confusing if an old reason was left in - so check leaving date is there.
-            $row["Leaving Reason"] = $reg->family->leaving_on ? $reg->family->leaving_reason : null;
+            $row['Leaving Reason'] = $reg->family->leaving_on ? $reg->family->leaving_reason : null;
 
-            if ($user->role === "foodmatters_user") {
-                $row["Date file was Downloaded"] = Carbon::today()->toDateString();
-            }
+            $row['Date file was Downloaded'] = Carbon::today()->toDateString();
```

Comment:
This works absolutely fine - you could also, just skip if 'Date file was Downloaded' is a member of the excludeColumns array.

Reply:
we can skip it? but should it not be created first? asign it a date value? adding it to the `$excludeColumns` just creates that column if it doesn't exist? and if so, how would it know about its date value? unless we have to do something else in the `exportRegistrationsSummary()` and nothing in the `getRegistrationsSummary()` 🤔 😕

Reply:
if the function has been told "skip showing this column" there's not much point in making it, just to get rid of it further down. 

unless i'm missing something?

Reply:
we do want to show that column to fm users, don't we? that's my understanding of this card. it excludes it for those who do not have the right to export the registrations. 
![Screenshot 2020-02-24 at 17 32 50](https://user-images.githubusercontent.com/29477363/75176261-bd569600-572b-11ea-920c-74c0065f0d8a.png)

Reply:
Yes, that is correct.